api/src/main/kotlin/com/gmtkgamejam/models/AuthTokenSet.kt

```diff
@@ -1,15 +1,17 @@
 package com.gmtkgamejam.models
 
+import java.util.*
```

Comment:
Doesn't Kotlin have its own Date type?

Reply:
The docs say yes, but my IDE says no 😬 
![image](https://user-images.githubusercontent.com/13312237/158025343-086027d8-169a-4d17-aa76-c6b6adb39ac5.png)

Reply:
Oh, apparently I'm reading the docs wrongly:



>>> Kotlin doesn't have its own date and time implementation. Why should the Kotlin developers focus on something Oracle created in [Java](https://www.ictdemy.com/java) 8? As we emphasized in the Kotlin basic course, Kotlin and Java are "compatible", they use the same virtual machine and that's why they can share their classes. Therefore, Kotlin uses Java classes for date and time.

[Source](https://www.ictdemy.com/kotlin/oop/date-and-time-in-kotlin-creating-and-formatting)